integration_tests/suite/test_cdr.py

```diff
@@ -1669,7 +1676,7 @@ def test_given_call_logs_when_list_cdr_of_user_recorded_filter_then_list_recorde
                 filtered=1,
                 total=3,
                 items=contains_inanyorder(
-                    has_entries(id=1),
+                    all_of(has_entries(id=1), not_(has_key('storage_path')))
```

Comment:
This check doesn't seem to work, since the `UserCDRSchemaList` is not used and everything pass